From 5175b1676bad982c9e526f7a0c8fd1893eef4ac5 Mon Sep 17 00:00:00 2001
From: Ting-Wei Lan <lantw@src.gnome.org>
Date: Sun, 26 Sep 2021 18:47:18 +0800
Subject: [PATCH] GDK/Win32: Use lowercase windows.h

Fix build on case-sensitive file systems, so it can be cross-compiled
from Linux with MinGW.

https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4000
---
 gdk/win32/gdkcairocontext-win32.c      | 2 +-
 gdk/win32/gdkhdataoutputstream-win32.c | 2 +-
 gdk/win32/gdkwin32cursor.h             | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/gdk/win32/gdkcairocontext-win32.c b/gdk/win32/gdkcairocontext-win32.c
index 9eb115ffcd..0f9dbbca1a 100644
--- a/gdk/win32/gdkcairocontext-win32.c
+++ b/gdk/win32/gdkcairocontext-win32.c
@@ -27,7 +27,7 @@
 
 #include <cairo-win32.h>
 
-#include <Windows.h>
+#include <windows.h>
 
 G_DEFINE_TYPE (GdkWin32CairoContext, gdk_win32_cairo_context, GDK_TYPE_CAIRO_CONTEXT)
 
diff --git a/gdk/win32/gdkhdataoutputstream-win32.c b/gdk/win32/gdkhdataoutputstream-win32.c
index 0075d0b66c..7fead7243d 100644
--- a/gdk/win32/gdkhdataoutputstream-win32.c
+++ b/gdk/win32/gdkhdataoutputstream-win32.c
@@ -20,7 +20,7 @@
 
 #include "config.h"
 
-#include <Windows.h>
+#include <windows.h>
 
 #include "gdkprivate-win32.h"
 #include "gdkhdataoutputstream-win32.h"
diff --git a/gdk/win32/gdkwin32cursor.h b/gdk/win32/gdkwin32cursor.h
index ec03f77212..a9f4676623 100644
--- a/gdk/win32/gdkwin32cursor.h
+++ b/gdk/win32/gdkwin32cursor.h
@@ -29,9 +29,9 @@
 #error "Only <gdk/gdkwin32.h> can be included directly."
 #endif
 
-#include <Windows.h>
 #include <gdk/gdk.h>
 #include <gdk/win32/gdkwin32display.h>
+#include <windows.h>
 
 G_BEGIN_DECLS
 
-- 
GitLab

